frontend/src/console/components/ConsolePage/ConsolePage.tsx

```diff
@@ -76,38 +75,6 @@ export class ConsolePageComponent extends React.PureComponent<Props> {
     }
   }
 
-  private renderHeader() {
-    if (this.props.loading || this.props.activeInstance.data === null) {
-      return (
-        <div className="title-container">
-          <h1>
-            <i className="fas fa-sync-alt fa-spin" />
-          </h1>
-        </div>
-      );
-    }
-
-    const instanceData = this.props.activeInstance.data;
-    const instanceLink = instanceData.lmsUrl;
-    const studioLink = instanceData.studioUrl;
-
-    return (
-      <div className="title-container">
-        <h1>
-          <a className="header-link" href={instanceLink}>
-            {instanceData.instanceName}
-            <i className="instance-link fas fa-link fa-xs" />
-          </a>
-        </h1>
-        <h2>
-          <a className="header-link" href={studioLink}>
-            <WrappedMessage messages={messages} id="editCourses" />
```

Comment:
If we are deleting this message, we should remove it from the `src/console/components/ConsolePage/displayMessages.ts` too.

Reply:
Deleted